script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -171,7 +172,7 @@ She is only ever used as a means for massacre.
 [sha:cd1a3d1b0c088f6eb59dfcf6b2721fc568462854]{
 -- Page 8, Offset 29004.
 -- 鮮血にまみれながら傷はなく。
-Drenched in blood, yet without wounds.
+Drenched in blood, yet entirely unscathed.
```

Comment:
hmm, I think unscathed here is kinda weird. It's not a very poetic word so if there's not a better version in arc route, just revert